### PR TITLE
Add Audio to Ubuntu

### PR DESCRIPTION
**Pull request by NoelHuibers**
Adds the Audiopackage to the Dockerimage and ReadMe.
Also updates packages.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -44,14 +44,15 @@ RUN apt-get update && \
...
-      python3=3.10.6-1~22.04 \
-      python3-dev=3.10.6-1~22.04 \
-      python3-pip=22.0.2+dfsg-1ubuntu0.4 \
+      python3=3.10.6-1~22.04.1 \
+      python3-dev=3.10.6-1~22.04.1 \
+      python3-pip=22.0.2+dfsg-1ubuntu0.5 \
       qt5-qmake=5.15.3+dfsg-2ubuntu0.2 \
       qtbase5-dev=5.15.3+dfsg-2ubuntu0.2 \
       xauth=1:1.1-1build2 && \
+      portaudio19-dev=19.6.0-1.1 && \
```
Comment (felixocker) [inline Dockerfile:55]: adding this line for portaudio requires removing the "&&" at the end of the previous line

**Review by felixocker — CHANGES_REQUESTED** [review]
thanks for the PR.
could you fix the issue in the Dockerfile or give me edit rights? then this should be good to go

**Comment by NoelHuibers** [comment]
Fixed it, good that you saw it.

**Comment by felixocker** [comment]
thanks!

**Pull request merged**